src/editor/sidebar/style.scss

```diff
@@ -60,3 +60,9 @@
 		margin: 0;
 	}
 }
+
+.interface-interface-skeleton__sidebar {
+	.components-notice {
+		margin: 0;
+	}
+}
```

Comment:
How can this be tested? The only notice in sidebar I can think of now is the color warning, but it already has a margin of 0.

![image](https://user-images.githubusercontent.com/7383192/80796653-11547f80-8ba0-11ea-8332-5cf70842a9b1.png)

Reply:
Oh yeah I forgot I added this. I can be seen when you send a newsletter with the success message.